Source/JavaScriptCore/wasm/WasmB3IRGenerator.cpp

```diff
@@ -4463,10 +4480,11 @@ auto B3IRGenerator::addCall(uint32_t functionIndex, const TypeDefinition& signat
     }
 
     if (callType == CallType::Call
+        && functionIndex - m_numImportFunctions != m_functionIndex
```

Comment:
What is the purpose of this?

Reply:
If we inline ourself recursively, then stack traces won't work using this method. At the same time though, inlining ourself recursively seems to me like it would rarely be helpful

Reply:
OK.